src/promptflow/promptflow/_utils/multimedia_utils.py

```diff
@@ -206,13 +206,17 @@ def _process_recursively(value: Any, process_funcs: Dict[type, Callable] = None)
     return value
 
 
-def load_multimedia_data(inputs: Dict[str, FlowInputDefinition], line_inputs: dict):
+def load_multimedia_data(
```

Comment:
I think we could just reverse dict here?
Just check `line_inputs`'s type and decide to convert or not